<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1228</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PCS Image Viewer</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="rotateLeftButton">
        <property name="toolTip">
         <string>Rotate Left</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="object-rotate-left"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="rotateRightButton">
        <property name="toolTip">
         <string>Rotate Right</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset theme="object-rotate-right"/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="horizontalSlider">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QScrollArea" name="scrollArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="ImageView" name="imageView">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>1208</width>
         <height>662</height>
        </rect>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1228</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent">
     <property name="title">
      <string>&amp;Recent</string>
     </property>
     <property name="icon">
      <iconset theme="document-open-recent"/>
     </property>
     <addaction name="actionon_recent"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="actionProperties"/>
    <addaction name="menuRecent"/>
    <addaction name="actionClearRecent"/>
    <addaction name="actionNext"/>
    <addaction name="actionPrevious"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>&amp;View</string>
    </property>
    <addaction name="actionRotate_Right"/>
    <addaction name="actionRotateLeft"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Qt"/>
    <addaction name="actionAbout_Application"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset theme="document-open"/>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="icon">
    <iconset theme="list-remove"/>
   </property>
   <property name="text">
    <string>&amp;Close</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>&amp;Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionProperties">
   <property name="icon">
    <iconset theme="document-properties"/>
   </property>
   <property name="text">
    <string>&amp;Properties</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="actionRotate_Left">
   <property name="text">
    <string>&amp;Rotate Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionRotate_Right">
   <property name="icon">
    <iconset theme="object-rotate-right"/>
   </property>
   <property name="text">
    <string>Ro&amp;tate Right</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionon_recent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>&amp;on recent</string>
   </property>
  </action>
  <action name="actionAbout_Application">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>A&amp;bout Image Viewer</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset theme="help-about"/>
   </property>
   <property name="text">
    <string>&amp;About Qt</string>
   </property>
  </action>
  <action name="actionRotateLeft">
   <property name="icon">
    <iconset theme="object-rotate-left"/>
   </property>
   <property name="text">
    <string>&amp;Rotate Left</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
  <action name="actionNext">
   <property name="icon">
    <iconset theme="go-next"/>
   </property>
   <property name="text">
    <string>&amp;Next</string>
   </property>
   <property name="shortcut">
    <string>Right</string>
   </property>
  </action>
  <action name="actionPrevious">
   <property name="icon">
    <iconset theme="go-previous"/>
   </property>
   <property name="text">
    <string>Pre&amp;vious</string>
   </property>
   <property name="shortcut">
    <string>Left</string>
   </property>
  </action>
  <action name="actionClearRecent">
   <property name="icon">
    <iconset theme="edit-clear"/>
   </property>
   <property name="text">
    <string>C&amp;lear Recent</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageView</class>
   <extends>QWidget</extends>
   <header>src/imageview.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
